sdk/monitor/monitor-query/src/timespanConversion.ts

```diff
@@ -0,0 +1,69 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { TimeInterval } from "./models/common";
```

Comment:
This conversions are probably fine but at first glance they seem a little too generic for just checking which kind of the TimeInterval union this value is. Personally I like to make things explicit and generify when we have to but this is just opinion so totally up to you whether you want to make it more specific to TimeInterval.

I do think that this should live next to other transformations if possible but that's also a nit